docs/source/ephys_at_swc/resources.md

```diff
@@ -0,0 +1,113 @@
+# Resources
+
+This section includes articles and videos providing
+background and technical detail on extracellular
+electrophysiology data preprocessing and analysis.
+
+This section is by no means exhaustive, please
+feel free to [get in contact](community.md) to suggest
+additions to this page.
+
+## General Introduction
+
+Below are a selection of papers that give a history
+and overview of the extracellular electrophysiology landscape:
+
+[Steinmetz NA et al. (2018). Challenges and opportunities for large-scale electrophysiology with Neuropixels probes. *Current Opinion in Neurobiology*.](https://pubmed.ncbi.nlm.nih.gov/29444488/)
+
+[Buccino AP et al. (2022). Spike sorting: new trends and challenges of the era of high-density probes. *Progress in Biomedical Engineering*.](https://iopscience.iop.org/article/10.1088/2516-1091/ac6b96/meta)
+
+[Rey HG et al. (2015). Past, present and future of spike sorting techniques. *Brain Research Bulletin*.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4674014/)
+
+[Carlson D et al. (2019). Continuing progress of spike sorting in the era of big data. *Current Opinion in Neurobiology*](https://pubmed.ncbi.nlm.nih.gov/30856552/)
+
+## Technical Introduction
+
+This section includes more technical resources on the different stages of
+extracellular electrophysiology analysis.
+
+A particularly useful resource is the
+[Neuropixels](https://www.ucl.ac.uk/neuropixels/courses) course, with their videos published online
+(e.g. [2023](https://www.ucl.ac.uk/neuropixels/training/2023-neuropixels-course)).
+While these are targeted towards Neuropixels users, they are
+valuable resources for any researcher
+approaching electrophysiology preprocessing and analysis.
+
+### Preprocessing
+
+The [IBL white paper](https://figshare.com/articles/online_resource/Spike_sorting_pipeline_for_the_International_Brain_Laboratory/19705522)
+contains a clearly written overview of common preprocessing steps. Similarly,
+[Bill Karsh's guide](https://billkarsh.github.io/SpikeGLX/help/catgt_tshift/catgt_tshift/) on
+SpikeGLX preprocessing tools gives a useful overview.
+
+[de Cheveigné & Nelken (2019)](https://pubmed.ncbi.nlm.nih.gov/30998899/)
+provide a technical treatment of digital filtering, a key step in preprocessing and analysis.
+
+### Spike Sorting
+
+[This video on Spike Sorting with Christophe Pouzat](https://www.youtube.com/watch?v=vSydfDvsewY),
+provides an excellent overview of the spike-sorting problem.
+
+[This chapter](https://neurophysics.ucsd.edu/publications/obd_ch3_2.pdf) provides a
+more detailed introduction to spike sorting and associated quality metrics.
```

Comment:
Might be worth adding more details about what this is? On my browser, this link just downloads a PDF.

Reply:
I'm not sure what to do about this one, it is a really fantastic article, but I can find absolutely no information about who it was written by or where it was published!